politeiawww/api/www/v1/v1.go

```diff
@@ -116,68 +114,68 @@ const (
-	ErrorStatusNoPublicKey                 ErrorStatusT = 22
-	ErrorStatusInvalidSignature            ErrorStatusT = 23
-	ErrorStatusInvalidInput                ErrorStatusT = 24
-	ErrorStatusInvalidSigningKey           ErrorStatusT = 25
-	ErrorStatusCommentLengthExceededPolicy ErrorStatusT = 26
-	ErrorStatusUserNotFound                ErrorStatusT = 27
-	ErrorStatusWrongStatus                 ErrorStatusT = 28
-	ErrorStatusNotLoggedIn                 ErrorStatusT = 29
-	ErrorStatusUserNotPaid                 ErrorStatusT = 30
-	ErrorStatusReviewerAdminEqualsAuthor   ErrorStatusT = 31
-	ErrorStatusMalformedUsername           ErrorStatusT = 32
-	ErrorStatusDuplicateUsername           ErrorStatusT = 33
-	ErrorStatusVerificationTokenUnexpired  ErrorStatusT = 34
-	ErrorStatusCannotVerifyPayment         ErrorStatusT = 35
-	ErrorStatusDuplicatePublicKey          ErrorStatusT = 36
-	ErrorStatusInvalidPropVoteStatus       ErrorStatusT = 37
-	ErrorStatusUserLocked                  ErrorStatusT = 38
-	ErrorStatusNoProposalCredits           ErrorStatusT = 39
-	ErrorStatusInvalidUserManageAction     ErrorStatusT = 40
-	ErrorStatusUserActionNotAllowed        ErrorStatusT = 41
-	ErrorStatusWrongVoteStatus             ErrorStatusT = 42
-	ErrorStatusCannotCommentOnProp         ErrorStatusT = 43
-	ErrorStatusCannotVoteOnPropComment     ErrorStatusT = 44
-	ErrorStatusChangeMessageCannotBeBlank  ErrorStatusT = 45
-	ErrorStatusCensorReasonCannotBeBlank   ErrorStatusT = 46
-	ErrorStatusCannotCensorComment         ErrorStatusT = 47
-	ErrorStatusUserNotAuthor               ErrorStatusT = 48
-	ErrorStatusVoteNotAuthorized           ErrorStatusT = 49
-	ErrorStatusVoteAlreadyAuthorized       ErrorStatusT = 50
-	ErrorStatusInvalidAuthVoteAction       ErrorStatusT = 51
-	ErrorStatusUserDeactivated             ErrorStatusT = 52
-	ErrorStatusInvalidPropVoteBits         ErrorStatusT = 53
-	ErrorStatusInvalidPropVoteParams       ErrorStatusT = 54
-	ErrorStatusEmailNotVerified            ErrorStatusT = 55
-	ErrorStatusInvalidUUID                 ErrorStatusT = 56
-	ErrorStatusInvalidLikeCommentAction    ErrorStatusT = 57
-	ErrorStatusInvalidCensorshipToken      ErrorStatusT = 58
-	ErrorStatusEmailAlreadyVerified        ErrorStatusT = 59
-	ErrorStatusInvalidPassword             ErrorStatusT = 85
-	ErrorStatusNoProposalChanges           ErrorStatusT = 88
+	ErrorStatusInvalid                         ErrorStatusT = 0
+	ErrorStatusInvalidEmailOrPassword          ErrorStatusT = 1
+	ErrorStatusMalformedEmail                  ErrorStatusT = 2
+	ErrorStatusVerificationTokenInvalid        ErrorStatusT = 3
+	ErrorStatusVerificationTokenExpired        ErrorStatusT = 4
+	ErrorStatusNoIndexFile                     ErrorStatusT = 5
+	ErrorStatusProposalNotFound                ErrorStatusT = 6
+	ErrorStatusProposalDuplicateFilenames      ErrorStatusT = 7
+	ErrorStatusProposalInvalidTitle            ErrorStatusT = 8
+	ErrorStatusMaxIndexFileExceededPolicy      ErrorStatusT = 9
```

Comment:
Can we trim the Policy off the end of these error messages?  It makes them super long and it's not really necessary.

Reply:
You're right, much better with the trimmed policy. Done.